services/consensuscontext/service.go

```diff
@@ -38,8 +41,7 @@ func (s *service) RequestNewTransactionsBlock(input *services.RequestNewTransact
 		return nil, err
 	}
 
-	// TODO: add reporting mechanism to this service and convert this print to it
-	fmt.Printf("created Transactions block num-transactions=%d block=%s\n", len(txBlock.SignedTransactions), txBlock.String())
+	s.reporting.Info("created Transactions block", instrumentation.Int("num-transactions", len(txBlock.SignedTransactions)), instrumentation.String("block", txBlock.String()))
```

Comment:
Why `txBlock.String()`? We have `instrumentation.Stringable`.

Reply:
Thanks, fixed.

Reply:
Thanks, fixed.